HISTORY.rst

```diff
@@ -2,6 +2,38 @@
 History
 =======
 
+3.39.0 (2021-07-08)
+-------------------
+Critical Changes:
+* A new Snowfakery task with better usability and multi-processor support. Look at the CumulusCI docs to learn the new syntax.
```

Comment:
Let's name the task here so users can go look it up

Reply:
@prescod is this just `snowfakery` task?